data/template.yml

```diff
@@ -0,0 +1,96 @@
+# State grade title to number mapping:
+#   Model state => 3
+#   Survivor wealth friendly => 2
+#   Taking steps => 1
+#   Some accountability => 0
+#   Little accountability => -1
+
+# Category grade title to number mapping:
+#   Model policy => 3
+#   Strong policy => 2
+#   Promising policy => 1
+#   No policy => 0
+#   Adverse policy => -1
```

Comment:
@sarahgreen This is a template file we can copy and paste for the other states. I included the comments above as a mapping between the grading labels used by FreeFrom and the underlying numbers in our system.